situp-core/src/main/java/com/amazon/situp/parser/model/PipelineConfiguration.java

```diff
@@ -132,4 +133,11 @@ private Integer getValueFromConfiguration(final Integer configuration, final Str
         }
         return configuration;
     }
+
+    /**
+     * TODO Implement this to use CPU cores of the executing machine
+     */
+    private int getDefaultProcessorThreads() {
+        return 1; //Runtime.getRuntime().availableProcessors()
```

Comment:
nit: Maybe create a static constant like delay?

Note: We were in a meeting and I was searching for this value :(